### PR TITLE
Issue #6214: Add CKE version to telemetry.

### LINKED ISSUE #6214: Update the telemetry module to report the CKEditor versions
In order to support removing ckeditor 4 from Backdrop core in version 2.x, we will need telemetry data to show how many sites have switched from ckeditor 4 to ckeditor 5.

We will need to report which major version(s) of CKeditor sites are using, including those that are using both versions at the same time. 

Implementation:
* The core `ckeditor` module will provide `ckeditor_version` data
* The contrib `ckeditor5` module will provide `ckeditor_version5` data
* The contrib `ckeditor5` module will provide `ckeditor_multiple` data, and test for the existance of ckeditor version 4.

### PR DESCRIPTION
**Pull request by jenlampton**
Fixes https://github.com/backdrop/backdrop-issues/issues/6214

### CODE REVIEW

File: core/modules/ckeditor/ckeditor.module [hunk 1]
```diff
@@ -777,3 +777,26 @@ function _ckeditor_theme_css($theme = NULL) {
...
   return $css;
 }
+
+/**
+ * Implements hook_telemetry_info().
+ */
+function ckeditor_telemetry_info() {
+  // If this were in mymodule.module, "project" would assume to be "mymodule".
+  $info['cke_version'] = array(
+    'label' => t('CKeditor Version'),
```
Comment (quicksketch) [inline core/modules/ckeditor/ckeditor.module]:
```suggestion
    'label' => t('CKEditor Version'),
```

File: core/modules/ckeditor/ckeditor.module [hunk 2]
```diff
@@ -777,3 +777,26 @@ function _ckeditor_theme_css($theme = NULL) {
   }
   return $css;
 }
+
+/**
+ * Implements hook_telemetry_info().
+ */
+function ckeditor_telemetry_info() {
+  // If this were in mymodule.module, "project" would assume to be "mymodule".
+  $info['cke_version'] = array(
```
Comment (quicksketch) [inline core/modules/ckeditor/ckeditor.module]: Maybe expand this info key. There's no reason to abbreviate.
```suggestion
  $info['ckeditor_version'] = array(
```

File: core/modules/ckeditor/ckeditor.module [hunk 3]
```diff
@@ -777,3 +777,26 @@ function _ckeditor_theme_css($theme = NULL) {
   }
   return $css;
 }
+
+/**
+ * Implements hook_telemetry_info().
+ */
+function ckeditor_telemetry_info() {
+  // If this were in mymodule.module, "project" would assume to be "mymodule".
```
Comment (quicksketch) [inline core/modules/ckeditor/ckeditor.module]:
```suggestion
```

File: core/modules/ckeditor/ckeditor.module [hunk 4]
```diff
@@ -777,3 +777,26 @@ function _ckeditor_theme_css($theme = NULL) {
...
+  $info['cke_version'] = array(
+    'label' => t('CKeditor Version'),
+    'description' => t('The current version of CKEditor in use.'),
+    'project' => 'telemetry',
```
Comment (quicksketch) [inline core/modules/ckeditor/ckeditor.module]: The "project" key determines which project node on BackdropCMS.org receives the telemetry data. Assuming we want this to show on https://backdropcms.org/project/backdrop/telemetry, this should be `backdrop`

```suggestion
    'project' => 'backdrop',
```

File: core/modules/ckeditor/ckeditor.module [hunk 5]
```diff
@@ -777,3 +777,25 @@ function _ckeditor_theme_css($theme = NULL) {
...
+  );
+  return $info;
+}
+
+/**
+ * Implements hook_telemetry_data().
+ */
+function ckeditor_telemetry_data($telemetry_key) {
+  switch ($telemetry_key) {
+    case 'cke_version':
```
Comment (bugfolder) [inline core/modules/ckeditor/ckeditor.module]:
```suggestion
    case 'ckeditor_version':
```

**Comment by quicksketch** [comment]
Merged into 1.x for 1.26.0. Thanks @jenlampton and @bugfolder!

**Pull request merged**